vandelay-py.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```

Comment:
should this be committed?

Reply:
Yeah I know it's weird, but it is an extension which enable to import automatically the missing variables. I use it a lot